src/cli/commands/force/lightning/lwc/test/ui/mobile/run.ts

```diff
@@ -0,0 +1,184 @@
+        `$ force:lightning:lwc:test:ui:mobile:run --config '/path/to/wdio.conf.js' --spec '/path/to/myTest.spec.js'`,
+        `$ force:lightning:lwc:test:ui:mobile:run --config '/path/to/wdio.conf.js' --spec '/path/to/folderWithSpecs'`
+    ];
+
+    private static createError(stringId: string): SfError {
+        return new SfError(
+            messages.getMessage(stringId),
+            'lwc-dev-mobile',
+            Run.examples
+        );
+    }
+
+    public static flagsConfig = {
+        config: flags.string({
+            description: messages.getMessage('configFlagDescription'),
+            char: 'f',
+            required: true
+        }),
+        spec: flags.string({
+            description: messages.getMessage('specFlagDescription'),
+            char: 's',
+            required: false
+        })
+    };
+
+    public async init(): Promise<void> {
+        if (this.logger) {
+            // already initialized
+            return Promise.resolve();
+        }
+
+        CommandLineUtils.flagFailureActionMessages = Run.examples;
+        return super
+            .init()
+            .then(() =>
+                Logger.child('force:lightning:lwc:test:ui:mobile:run', {})
+            )
+            .then((logger) => {
+                this.logger = logger;
+                return Promise.resolve();
+            });
+    }
+
+    public async run(): Promise<any> {
+        this.logger.info(`UTAM run command invoked.`);
+
+        const config = CommandLineUtils.resolveFlag(this.flags.config, '');
+        const configPath = path.normalize(
+            path.resolve(CommonUtils.resolveUserHomePath(config))
+        );
+        if (!fs.existsSync(configPath)) {
+            CommonUtils.stopCliAction();
+            return Promise.reject(
+                Run.createError('error:configFileDoesntExistDescription')
+            );
+        }
+
+        let specs = '';
+        const spec = CommandLineUtils.resolveFlag(this.flags.spec, '');
+        if (spec.length > 0) {
+            const specPath = path.normalize(
+                path.resolve(CommonUtils.resolveUserHomePath(spec))
+            );
+            if (!fs.existsSync(specPath)) {
+                CommonUtils.stopCliAction();
+                return Promise.reject(
+                    Run.createError('error:specPathInvalidDescription')
+                );
+            }
+            let specsArray = this.enumerateTestSpecs(specPath);
+            specsArray = specsArray.map((spec) => `'${spec}'`);
+            specs = specsArray.join(' ');
+        }
+
+        CommonUtils.startCliAction(messages.getMessage('runningUtamTest'));
+        return await this.executeRunUtamTest(configPath, specs)
+            .then((result) => {
+                CommonUtils.stopCliAction();
+
+                this.logger.info(
+                    `UTAM test ran successfully:\n${result.stdout}`
+                );
+                console.log(result.stdout);
```

Comment:
I have `console.log` here because without it Appium test would run but not report back anything back to the terminal. Ideally we have a formatted report in the end.

Reply:
I think `console.log` will cause ESLint violation of rule `no-console` so at the very least you'd need to add `// tslint:disable-next-line: no-console` on the line before.